spec/components/provider_interface/application_timeline_component_spec.rb

```diff
@@ -224,6 +224,17 @@ def provider_user
         expect(rendered.css('.app-timeline__actor_and_date').text).to include 'System'
       end
     end
+
+    context 'when change was done by the vendor api' do
+      let(:user) { create(:vendor_api_user) }
+
+      it 'exposes the vendor Api users name' do
+        application_choice = application_choice_with_audits [audit]
```

Comment:
Looks great. Maybe this can be moved into a let block of the describe block so it can be reused across the nested examples, as it seems to be identical for all scenarios.